ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiServerConfig.scala

```diff
@@ -24,14 +24,14 @@ case class ApiServerConfig(
     apiStreamShutdownTimeout: Duration = ApiServerConfig.DefaultApiStreamShutdownTimeout,
     command: CommandConfiguration = ApiServerConfig.DefaultCommand,
     configurationLoadTimeout: Duration = ApiServerConfig.DefaultConfigurationLoadTimeout,
-    initialLedgerConfiguration: Option[InitialLedgerConfiguration] =
+    initialLedgerConfiguration: InitialLedgerConfiguration =
       ApiServerConfig.DefaultInitialLedgerConfiguration,
     managementServiceTimeout: FiniteDuration = ApiServerConfig.DefaultManagementServiceTimeout,
     maxInboundMessageSize: Int = ApiServerConfig.DefaultMaxInboundMessageSize,
     party: PartyConfiguration = ApiServerConfig.DefaultParty,
     port: Port = ApiServerConfig.DefaultPort,
     portFile: Option[Path] = ApiServerConfig.DefaultPortFile,
-    rateLimitingConfig: Option[RateLimitingConfig] = ApiServerConfig.DefaultRateLimitingConfig,
+    rateLimit: RateLimitingConfig = ApiServerConfig.DefaultRateLimitingConfig,
```

Comment:
Before the optionality of this was a way of turning off the rate limiting config, that may still be needed.